src/store/trees/trees.js

```diff
@@ -162,19 +160,24 @@ export default {
 
   mutations: {
 
+    // TODO: find a better place to put the configure, putting it inside trees store doesn't make sense
```

Comment:
I think we need to think about this, and create a ticket if needed. What do you think ?

Reply:
@safydy We can tackle this once we can remove this trees store from flow-builder. I think that will shake out together, I don't envision us always having to do this funky business in the flow-builder repo. :) 

For now, I think we'll get by and I will create a JIRA 👌 Thanks for the note!

Reply:
We now have: https://viamoinc.atlassian.net/browse/VMO-3577

Reply:
Got that, thanks for the ticket ;)